### PR TITLE
Fix broken sections

### PR DESCRIPTION
**Pull request by annda**
Previous implementation broke sections and section editing

### CODE REVIEW

File: syntax/closesection.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+if(!defined('DOKU_INC')) die();
+
+if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
+require_once(DOKU_PLUGIN.'syntax.php');
```
Comment (splitbrain) [inline syntax/closesection.php]: these three lines can be removed (PSR-2)

File: syntax/closesection.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+/**
+ * Copy of syntax_plugin_wrap_closesection
+ * Used for special handling of headers
+ */
+class syntax_plugin_ifauthex_closesection extends DokuWiki_Syntax_Plugin {
```
Comment (splitbrain) [inline syntax/closesection.php]: reformat for PSR-2 please

File: syntax/closesection.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    function getType() { return 'substition';}
+    function getPType() { return 'block';}
+    function getSort() { return 195; }
+
+    /**
+     * Dummy handler, this syntax part has no syntax but is directly added to the instructions by the div syntax
+     */
```
Comment (splitbrain) [inline syntax/closesection.php:14]: comment seems to be outdated.

**Review by splitbrain — APPROVED** [review]
can be merged after the minor updates

File: syntax/closesection.php (hunk 1 above)
Comment (annda) [reply syntax/closesection.php]: fixed

File: syntax/closesection.php (hunk 2 above)
Comment (annda) [reply syntax/closesection.php]: fixed

File: syntax/closesection.php (hunk 3 above)
Comment (annda) [reply syntax/closesection.php:14]: fixed

**Comment by annda** [comment]
Sorry, I'm closing this as a duplicate of #3

**Pull request closed without merging**